components/sensor/cd74hc4067.rst

```diff
@@ -0,0 +1,87 @@
+cd74hc4067 Analog Multiplexer
+=============================
+
+.. seo::
+    :description: Instructions for setting up cd74hc4067 Analog Multiplexer and sensors.
+    :image: cd74hc4067.jpg
+
+
+The ``cd74hc4067`` Analog Multiplexer component allows you to attach up to 16 analog input sensors to a single ADC pin.
+The component is used to activate the right multiplexer pin for each of the defined cd74hc4067 sensors.
+Since the cd74hc4067 sensors are also voltage sensors, they can be used as input for :doc:`CT clamp sensors <ct_clamp>`, 
+as in the configuration example below.
+
+For further information about the cd74hc4067 chip see the `spec sheet  <https://www.ti.com/lit/ds/symlink/cd74hc4067.pdf>`__
+and there are also some guides around like the one from `Adam Meyer <http://adam-meyer.com/arduino/CD74HC4067>`__.
+
+Actually, the chip is bidirectional and could also be used as output, but this is not supported by the component yet.
+
+First, you need to configure the component with the digital pins that control the multiplexer. 
+Then you need to set up a voltage sensor source (e.g. :doc:`ADC sensor <adc>`) and pass it to the cd74hc4067 sensors with the ``sensor`` option.
+Each cd74hc4067 sensor is configured for one of the 16 input pins of the multiplexer.
+
+In the example below, the ``adc`` and ``cd74hc4067`` sensors are internal and :doc:`CT clamp sensors <ct_clamp>` used on each of the sensors.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    cd74hc4067:
+    pin_s0: D0
+    pin_s1: D1
+    pin_s2: D2
+    pin_s3: D3     
+
+    sensor:
+    - platform: adc
+        id: adc_sensor
+        pin: A0
+        update_interval: 3600s
+    - platform: cd74hc4067
+        id: solar_1_raw
+        number: 0
+        sensor: adc_sensor
+        update_interval: 3600s
+    - platform: cd74hc4067
+        id: solar_2_raw
+        number: 1
+        sensor: adc_sensor
+        update_interval: 3600s
+    - platform: ct_clamp
+        name: "Solar1"
+        sensor: solar_1_raw
+        update_interval: 5s
+        unit_of_measurement: "A"
+        filters:
+            - multiply: 36000
+    - platform: ct_clamp
+        name: "Solar2"
+        sensor: solar_2_raw
+        update_interval: 5s
+        unit_of_measurement: "A"
+        filters:
+            - multiply: 36000
```

Comment:
While I appreciate trying to help people with examples, the config block here should be specific to this component only and basically a bare minimum to get it working.
Please also remove the ct_clamp references above to make it a quicker read.

If you want to show a fuller example like this then you can make another sub-heading before `See Also` with the details.

Reply:
The configuration is reduced to the minimum and an extra application example added.